overview/intro.md

```diff
@@ -2,67 +2,113 @@
+## Benefits of Prebid Products
 
-Header Bidding is a technique that involves running SSP & Ad Exchange code directly on page so publishers can receive bids on their inventory that may be unavailable through their primary ad server and exchange.
+The Prebid product suite offers publishers multiple benefits designed to foster a better header bidding experience:
+1. Free and open source, enabling anyone to contribute or review code.
+2. The largest repository of working header bidding adapters.
+3. Asynchronous and single time-out to provide a better user experience.
+4. Prebid Server to run faster auctions with more partners.
+5. Tools and analytics to optimize your setup.
+6. Multiple options on formats (display, video, native) and channels (mobile, web).
+7. A well established and helpful community which can advise on best practices and if needed provide professional setup and services.  
 
-The returned bids are then passed into the ad server so they can compete with direct demand and the primary ad server's exchange on a level playing field.
+## Prebid Products
 
-## A Brief History of Header Bidding
+### Prebid.js
 
-The early days of Header Bidding were dominated by bad practices, closed proprietary tech, poor standards, and little to no cooperation between competing companies. The result for publishers was headache - having to manually patch together various solutions from different companies and hope their developer didn't spend 3 weeks coding the wrong thing.
+Prebid.js is the core product of the Prebid suite. Implemented on multiple formats to include display, video, and native, it provides a simple process for header bidding that can be ramped up to fit the complexity of your needs.
 
-Launched in 2015, Prebid.js changed the game and made header bidding easy for publishers. By creating a simple, open tech layer upon which companies could add their code to a standard but optimized foundation, Prebid.js made it easier to implement header bidding the right way, and offered the largest repository of working adapters.
+**Reducing Latency**
 
-Today, Prebid.js is the most widely used Header Bidding "container" or "wrapper" on the web. The ecosystem supports 150+ demand partners, 15+ analytics providers, and numerous publishers.
+One of the main problems publishers experienced with other header bidding solutions was the delay between the bid requests being sent and the responses being returned. Many solutions use synchronous calls, meaning each SSP or ad exchange had to receive the request and return a response before the next SSP was called. This could lead to long delay from the web page being called to it loading. 
+Prebid resolves this issue by concurrently calling the selected SSPs and ad exchanges within the set timeout. That setting is respected by Prebid.js and any bidder not returning a result within the timeout duration is excluded from the auction. This dramatically decreases the page load time, providing a better user experience. 
```

Comment:
add a comma after Prebid.js:  "That setting is respected by Prebid.js, and any bidder..."

Reply:
Added the comma.